docs/observability/how_to_guides/tracing/mask_inputs_outputs.mdx

```diff
@@ -292,3 +292,317 @@ async def async_function(key: str) -> int:
 ```
 
 These function-level processors take precedence over client-level processors (`hide_inputs` and `hide_outputs`) when both are defined.
+
+## Quick starts
+
+You can combine LangSmith's rule-based masking with various anonymizers to scrub sensitive information from inputs and outputs. In this how-to-guide, we'll cover working with Microsoft Presidio, AWS Comprehend, and regex.
```

Comment:
idk if i'd box langsmith into "rule-based masking"

Reply:
Maybe start with regex since that illustrates the technique also / doesn't require onboarding

Reply:
great points